featurebyte/service/feature_list_status.py

```diff
@@ -63,18 +63,28 @@ async def update_feature_list_namespace_status(
             # if no change in status, do nothing
             return
 
+        if (
+            feature_list_namespace.status == FeatureListStatus.DRAFT
+            and target_feature_list_status == FeatureListStatus.DEPRECATED
+        ):
+            raise DocumentUpdateError(
+                "Not allowed to update status of feature list from DRAFT to DEPRECATED. "
+                "Valid status transition: DRAFT -> PUBLIC_DRAFT or DRAFT -> TEMPLATE. "
+                "Please use delete feature list instead if it is no longer needed."
```

Comment:
Is there an extra word `use` in the last sentence? Maybe can update to:

```
Please delete the feature list instead if it is no longer needed.
```

Reply:
Good catch, will update this.